cypress/e2e/dashboard/chapters/chapter-administrator.cy.ts

```diff
@@ -0,0 +1,237 @@
+import { User } from '../../../../cypress.config';
+import { expectNoErrors } from '../../../support/util';
+
+const firstChapterId = 1;
+const secondChapterId = 2;
+
+describe('Chapter Administrator', () => {
+  let chapterRoles;
+  let instanceRoles;
+  let users;
+  before(() => {
+    cy.fixture('chapterRoles').then((fixture) => {
+      chapterRoles = fixture;
+    });
+    cy.fixture('instanceRoles').then((fixture) => {
+      instanceRoles = fixture;
+    });
+    cy.fixture('users').then((fixture) => {
+      users = fixture;
+    });
+  });
```

Comment:
I'm not sure this will necessarily cause issues, but the Cypress docs discourage use of variables like this.  The reasoning is that `cy.whatever` calls are generally asynchronous and so can cause fun bugs if you access them in places they're not guaranteed to exist.

Which is all to say: this code is probably fine, but not a best practice.  Could you refactor it to use aliases instead?  https://docs.cypress.io/guides/core-concepts/variables-and-aliases#Aliases covers this particular use-case quite well.

Reply:
I'm not entirely sure, but this might be slight exception https://docs.cypress.io/api/commands/fixture is linking to two pages using this approach https://glebbahmutov.com/blog/fixtures-in-custom-commands/ and https://github.com/cypress-io/testing-workshop-cypress/tree/master/cypress/integration/14-fixtures

I've started to change for using aliases and to be honest that's going to be painful. There's couple reasons for that:
- Resetting of aliases after each test, so fixtures would need to be loaded in the `beforeEach`. 
- As we have three fixtures here, that's three `cy.get('@alias').then(...)` in each test.
- Additionally to using `cy.get`, object with fixture passed to `.then` method doesn't have correct type by default.

What do you think?